tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -0,0 +1,25 @@
+export class God {
+  static create() {
+    return [new Man('Adam'), new Woman('Eve')];
+  }
+}
+
+export class Human {
+  constructor(name) {
+    this.name = name;
+  }
+}
+
+export class Man extends Human {
+  constructor( name, sex ) {
```

Comment:
as every `Human` has sex, it would be better to move this property to `Human` class instead and pass it to base class using super from `Man` and `Woman`

Reply:
Refactored